oneview_redfish_toolkit/tests/blueprints/test_computer_system.py

```diff
@@ -458,3 +458,72 @@ def test_change_power_state_invalid_key(self):
         )
         self.assertEqual("application/json", response.mimetype)
         self.assertEqual(result, invalid_json_key)
+
+    @mock.patch.object(computer_system, 'g')
```

Comment:
It is a bit odd that this test is working, as we are not setting any behavior in the oneview_client mock. I was expecting to see a `g.oneview_client.server_profiles.delete.return_value = True`. Why is it not necessary?

Reply:
Because I was not checking if the response was True, just handling if it was not False. Once it was a mock object, it was different from False. But now the code is explicitly checking for True.